app/src/main/java/com/github/sdp/mediato/ui/viewmodel/SearchUserViewModel.java

```diff
@@ -1,68 +1,9 @@
 package com.github.sdp.mediato.ui.viewmodel;
 
-import androidx.lifecycle.LiveData;
```

Comment:
I'm not sure I understand the changes here. Why is all the file gone ?

Reply:
Nop there is still the loadUser() that is an abstract function of UserModelView